ffi/dotnet-idsdk/IoticsIdLib/src/Identity.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System.Net;
+
+namespace IOTICS;
+
+// Custom RuntimeException class
+public class IoticsLibException(string message) : ApplicationException(message)
+{
+}
+
+public class DelegationException(string message) : IoticsLibException(message) {
+    
+}
+
+/** 
+Factory class for identity. 
+It takes a resolver address as discovered by https://<your_space>.iotics.space/index.json
```

Comment:
I'm not sure this belongs here - it's not mentioned anywhere else in this repo yet.

Reply:
I have now mentioned in the README

Reply:
also - it's a true statement but it doesn't exclude any other source for the resolver address (do we have others?)